### PR TITLE
rabbitmq server 

### PR DESCRIPTION
**Pull request by scopatz**
Yet another attempt

### CODE REVIEW

**Comment by jakirkham** [comment]
So, I guess add `erlang` as a build and run dependency.

**Comment by scopatz** [comment]
Yeah, just waiting for erlang to be uploaded to anaconda.org

**Comment by scopatz** [comment]
There is a large standard deviation on the upload times, for reasons I don't understand.

**Comment by jakirkham** [comment]
I see I guess we are still waiting on Linux.

**Comment by jakirkham** [comment]
Weird. So CircleCI finished testing, but never tried to upload. Restarting...

**Comment by scopatz** [comment]
Well, sometimes it takes a long time ~2 - 3 hours.

**Comment by jakirkham** [comment]
What?! Really? Could you give me an example?

In this case, it actually finished, but didn't do what it was suppose.

**Comment by scopatz** [comment]
I don't remember which packages I have seen this with. But I'll let you know explicitly if it happens again.  Note that it took ~ 1 hour for the mac package of erlang to upload to anaconda.org after the merge happened.

**Comment by jakirkham** [comment]
So, one thing that can happens is the CI builds could wait awhile to begin. That can cause issues. Unfortunately this is not something we have control over. It does appear to be the case here with Travis CI that it waited ~40mins. Why it waited that long I don't know. Though I have generally found that when US East Coast and US West Coast work times overlap things are worse (particularly on Travis CI). For instance, 12P ET is 9A ET. So normally, I have found the afternoon by East Coast time is a bit rough to get things built on Travis CI. It appears Circle CI has Mac support now, but I don't know what they offer. I suppose we could evaluate that alternative.

**Comment by jakirkham** [comment]
Weird still not uploading. Added this issue ( https://github.com/conda-forge/erlang-feedstock/issues/1 ).

**Comment by jakirkham** [comment]
Toggling to see the present state of CIs.

**Comment by jakirkham** [comment]
Though I expect this still needs a merge/rebase with current `master`. Mainly for CircleCI if nothing else.

File: recipes/rabbitmq-server/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  version: {{ version }}
+
+source:
+  fn: rabbitmq-server-{{ platform }}-{{ version }}.tar.xz
+  url: http://www.rabbitmq.com/releases/rabbitmq-server/v{{ version }}/rabbitmq-server-{{ platform }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
```
Comment (jakirkham) [inline recipes/rabbitmq-server/meta.yaml]: Doesn't this depend on `erlang`?

**Comment by scopatz** [comment]
Closing as I am no longer interested. Please feel free to use the recipe, if you want.

**Comment by jakirkham** [comment]
FWIW PR ( https://github.com/conda-forge/staged-recipes/pull/2867 ) appears to have taken on adding this package.

**Pull request closed without merging**